docs/install/docker-plugin.rst

```diff
@@ -0,0 +1,74 @@
+
+========================================
+Installing the Flocker Plugin for Docker
+========================================
+
+Before installing the Flocker plugin for Docker, you will need to have installed Flocker on some nodes, using the instructions in the previous topics.
+
+On the same machine where you ran ``flocker-ca`` while installing Flocker, you will need to generate a new API user certificate and key for a user named ``plugin``, as described in the :ref:`generate-api` instructions.
```

Comment:
:question: would it be easier here to give an example in a prompt section of the `flocker-ca` command to run? e.g.

```
flocker-ca create-api-certificate plugin
scp ./plugin.crt root@172.16.255.251:/etc/flocker/plugin.crt
scp ./plugin.key root@172.16.255.251:/etc/flocker/plugin.key
```

perhaps worth saying more clearly that Flongle requires access to the Flocker REST API, therefore you will need to create an API certificate and install it on each node you are also using the plugin.